app/controlplane/api/workflowcontract/v1/crafting_schema.proto

```diff
@@ -40,6 +40,11 @@ message CraftingSchema {
   // Policy groups to apply to this schema
   repeated PolicyGroupAttachment policy_groups = 7;
 
+  // Optional path to the workflow file content
+  string workflow_file = 8;
+  // Whether the runner is hosted or not
+  bool is_hosted_runner = 9;
```

Comment:
I think these properties should be put in the CraftingState instead (crafting_state.proto), since it represents the current runtime data. CraftingSchema is meant to be used as a schema spec.

Reply:
Thank you @jiparis - reworked. Appreciate that!